hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -54,11 +53,6 @@ protected ServiceName getParentServiceName(PathAddress parentAddress) {
     }
 
     @Override
-    protected boolean isResourceServiceRestartAllowed(OperationContext context, ServiceController<?> service) {
```

Comment:
I'm not sure if we could replicate what the `reload-required` does:
Mark the agent in a `restart-required` state and notify in every operation against the agent that a restart is required.